src/blocking/spi.rs

```diff
@@ -18,6 +36,16 @@ pub trait Write<W> {
     fn write(&mut self, words: &[W]) -> Result<(), Self::Error>;
 }
 
+/// Blocking read
+pub trait Read<W> {
+    /// Error type
+    type Error;
+
+    /// Reads `words` to the slave. The word value sent on MOSI during
+    /// reading is implementation defined, typically `0x00`, `0xFF`, or configurable.
+    fn read(&mut self, words: &mut [W]) -> Result<(), Self::Error>;
```

Comment:
Also, I'd suggest that this take `&mut [MaybeUninit<Word>]`.

Reply:
Agreed it'd be nice! However, it should be on all "read" traits, and on its own RFC/PR so it can be discussed properly independently.

Reply:
Ah, gotcha. Makes sense.

Reply:
discussion in #288